modules/cli/src/main/resources/META-INF/native-image/org.virtuslab/scala-cli-core/reflect-config.json

```diff
@@ -793,5 +793,153 @@
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"com.google.cloud.tools.jib.image.json.ContainerConfigurationTemplate$HealthCheckObjectTemplate",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"com.google.cloud.tools.jib.image.json.ContainerConfigurationTemplate$RootFilesystemObjectTemplate",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"com.google.cloud.tools.jib.image.json.DescriptorDigestDeserializer",
+    "methods":[{"name":"<init>","parameterTypes":[] }]
+  },
+  {
+    "name":"com.google.cloud.tools.jib.image.json.DescriptorDigestSerializer",
+    "methods":[{"name":"<init>","parameterTypes":[] }]
+  },
+  {
+    "name":"com.google.cloud.tools.jib.image.json.HistoryEntry",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"com.google.cloud.tools.jib.image.json.ImageMetadataTemplate",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"com.google.cloud.tools.jib.image.json.ManifestAndConfigTemplate",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"com.google.cloud.tools.jib.image.json.ManifestTemplate",
+    "allDeclaredMethods":true
+  },
+  {
+    "name":"com.google.cloud.tools.jib.image.json.V22ManifestListTemplate",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"com.google.cloud.tools.jib.image.json.V22ManifestListTemplate$ManifestDescriptorTemplate",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"com.google.cloud.tools.jib.image.json.V22ManifestListTemplate$ManifestDescriptorTemplate$Platform",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"com.google.cloud.tools.jib.image.json.V22ManifestTemplate",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"com.google.cloud.tools.jib.json.JsonTemplate",
+    "allDeclaredMethods":true
+  },
+  {
+    "name":"com.google.cloud.tools.jib.registry.RegistryAuthenticator$AuthenticationResponseTemplate",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors":true
+  },
+  {
+    "name":"com.google.api.client.http.GenericUrl",
+    "allDeclaredFields":true
+  },
+  {
+    "name":"com.google.api.client.http.HttpHeaders",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true
+  },
+  {
+    "name":"com.google.api.client.util.GenericData",
+    "allDeclaredFields":true
+  },
+  {
+    "name":"java.lang.String"
+  },
+  {
+    "name":"java.util.Date"
+  },
```

Comment:
These are usually not be necessary, unless gson uses those at some point…

Reply:
If I remove `"name":"java.lang.String"`, then it throws following error:
```
Exception in thread "main" java.util.concurrent.ExecutionException: java.lang.NoClassDefFoundError: java.lang.String
        at com.google.common.util.concurrent.AbstractFuture.getDoneValue(AbstractFuture.java:566)
```